vineyard-bloom/vineyard-server-template
# Yeoman Lawn Testing

- Solidifying template files for common and Lawn projects
- Manually testing Yeoman Lawn project generation
- Updating automated test for Yeoman Lawn project generation
- Creating script for sample data generation
- Adding health check endpoints to generated API
- Clarifying setup instructions in readme
- Adding conditional rendering to Yeoman-generated readme
